index.rst

```diff
@@ -1,12 +1,9 @@
-Welcome to Zammad's documentation!
-==================================
-The Zammad documentation consists of three parts:
+System Documentation
+====================
 
-- Zammad system installation and configuration (this documentation)
-
-- Zammad administration (https://admin-docs.zammad.org)
-
-- Zammad user documentation (https://user-docs.zammad.org)
+.. hint::
+   You are currently reading the Zammad system documentation. There are also
+   administration_ and `user manuals`_ available.
```

Comment:
Why does the first link not need backticks?

Reply:
Because it is just one word. But I can add them to unify it.